Sources/Router/router.swift

```diff
@@ -31,9 +32,9 @@ public class Router {
                     }
                 }
             }
-            return generate404Response()
+            return generateResponse(version: "HTTP/1.1", statusCode: 404, statusPhrase: "NotFound", headers: ["Content-Length":String(errorMessage.count), "Content-Type":"text/html"], body: errorMessage)
```

Comment:
I kind of liked how these lines looked before you made the changes. `generate404Response` was easy to read and descriptive. Maybe create a private method call `fourOhFourResponse` that encapsulates this logic?